data/human/human missions.txt

```diff
@@ -3996,3 +3996,89 @@ mission "Fumbling Mugger"
 event "fumbling mugger followup timer"
 
 
+
+mission "Laborer"
+	name "Out of Work"
+	description "A man down on his luck needs to get to <destination>. He has promised to send you payment as soon as he has it."
+	passengers 1
+	source 
+		attributes "factory" "mining"
+	destination attributes "factory" "mining"
+	minor
+	to offer
+		random < 15
+	on offer
+		conversation
+			`Upon entering the spaceport, you see a young man in laborer's clothing is arguing with a woman in a flight suit.`
+			choice
+				`	(Go see what the laborer is arguing about).`
+				`	(Leave).`
+					decline
+			`	"Lady, I'm not trying to low-ball you here!" The man is nearly shouting. "I don't have the money! That's the whole point!"`
+			`	"I appreciate your conundrum," replies the woman, her voice clipped. "However, my policy is cash upfront. Good day."`
+			`	She turns and walks away briskly, and the man slumps against the bar.`
+			choice
+				`	"What was that all about?"`
+				`	(Leave before he notices you).`
+					decline
+			`	He gives you a once-over, and you see a spark of hope in his eyes. "You're a captain? Please, I need to get to <destination>. Work has dried up here, and I need a fresh start."`
+			choice
+				`	"And what's your offer?"`
+					goto offer
+				`	"Work dried up on the whole planet?"`
+			`	"This planet is messed up. Everybody likes to talk behind your back. I did a job for one of the big mining companies here, but my boss had it out for me. Told everybody I did it wrong, even though they never gave me any training at all. Then he went and put me on the permanent record. So yeah, sooner I get off this dump, the better."`
+			label offer
+			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
+			choice
+				`	"Okay, I can take you to <planet>."`
+					goto accept
+				`	"How do I know you'll pay?"`
+					goto question
+				`	"Sorry, money up front."`
+					goto decline
+			label question
+				`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
+			choice
+				`	"Sorry, money up front."`
+					goto decline
+				`	"Okay, I can take you to <planet>."`
+			label accept
+			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress."`
+				accept
+			label decline
+				`	"I knew it. You're just like everyone else."`
+					decline
+	on complete
+		conversation
+			`The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`
```

Comment:
Avoid telling the player what they're supposed to be thinking.

Reply:
I'm not really sure how to alter this so the meaning is intact while avoiding telling the player what they're thinking. Do you have any suggestions?